daemon/cmd/ipam.go

```diff
@@ -174,6 +174,18 @@ func (d *Daemon) DumpIPAM() *models.IPAMStatus {
 	return status
 }
 
+func (d *Daemon) allocateRouterIP(family datapath.NodeAddressingFamily) (net.IP, error) {
+	if option.Config.IPAM == "" && option.Config.LocalRouterIP != "" {
```

Comment:
If `IPAM` is set, it will silently ignore `LocalRouterIP`. Maybe best to throw an error in `daemon_main.go` if both flags are set?

Reply:
Good point! Added a check in daemon_main, removed check here. PTAL :)